spec/std/socket/addrinfo_spec.cr

```diff
@@ -24,8 +24,8 @@ describe Socket::Addrinfo, tags: "network" do
     end
 
     it "raises helpful message on getaddrinfo failure" do
-      expect_raises(Socket::Addrinfo::Error, "Hostname lookup for badhostname failed: ") do
-        Socket::Addrinfo.resolve("badhostname", 80, type: Socket::Type::DGRAM)
+      expect_raises(Socket::Addrinfo::Error, "Hostname lookup for badhostname.unknown failed: ") do
+        Socket::Addrinfo.resolve("badhostname.unknown", 80, type: Socket::Type::DGRAM)
```

Comment:
Oh, I didn't mean to push this fix in this PR but to make it it's own PR.

The problem here is that within my vagrant/libvirt VM resolving a hostname without an extension leads to 30 seconds (freeds, netbsd) to 2.5 minutes (openbsd) time to fail to resolve the name 😲

Using a domain with an extension, however, fails to resolve immediately 🤷